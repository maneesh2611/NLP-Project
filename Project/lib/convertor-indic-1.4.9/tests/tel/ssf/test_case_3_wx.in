1	sUpar	unk	<fs af='sUpar,n,,sg,,d,0,0'>|<fs af='sUpar,n,,sg,,o,ti,ti'>
2	stAr	unk	<fs af='stAr,n,,sg,,d,0,0'>|<fs af='stAr,n,,sg,,o,ti,ti'>
3	nane	unk	<fs af='nana,n,,sg,,,0,0_e'>
4	ahaM	unk	<fs af='ahaM,n,,sg,,d,0,0'>
5	lekuMdA	unk	<fs af='lekuMdA,avy,,,,,0,0_avy'>|<fs af='lekuMdu,v,any,any,any,,i_A,i_A'>
6	Ayana	unk	<fs af='Ayana,pn,,sg,,d,0,0'>|<fs af='Ayana,pn,,sg,,o,ti,ti'>
7	cUpina	unk	<fs af='cUpu,v,any,any,any,,ina,ina'>
8	ApyAyawaku	unk	<fs af='ApyAyawa,n,,sg,,,ki,ki'>
9	kaxilina	unk	<fs af='kaxulu,v,any,any,any,,ina,ina'>
10	poyina	unk	<fs af='poyina,n,,sg,,d,0,0'>|<fs af='poyina,n,,sg,,o,ti,ti'>|<fs af='po,v,any,any,any,,ina,ina'>
11	yUnit	unk	<fs af='yUnit,n,,sg,,d,0,0'>|<fs af='yUnit,n,,sg,,o,ti,ti'>
12	saByulu	unk	<fs af='saByudu,n,,pl,,d,0,0'>|<fs af='saByudu,n,,pl,,d,vu,vu'>
13	rajanI	unk	<fs af='rajani,n,,sg,,,0,0_V'>
14	,	unk	<fs af=',,punc,,,,,'>
15	Ayana	unk	<fs af='Ayana,pn,,sg,,d,0,0'>|<fs af='Ayana,pn,,sg,,o,ti,ti'>
16	kutuMba	unk	<fs af='kutuMbaM,n,,sg,,o,ti,ti'>
17	saByulu	unk	<fs af='saByudu,n,,pl,,d,0,0'>|<fs af='saByudu,n,,pl,,d,vu,vu'>
18	paxikAlAlu	unk	<fs af='paxikAlAlu,unk,,,,,,'>
19	pAtu	unk	<fs af='pAtu,n,,sg,,d,0,0'>|<fs af='pAtu,n,,sg,,o,ti,ti'>
20	callagA	unk	<fs af='callagA,avy,,,,,0,0_avy'>|<fs af='callagA,avy,,,,,0,0_avy'>|<fs af='calla,n,,sg,,,gA,gA'>|<fs af='calla,adj,,sg,,,gA,gA_adj'>|<fs af='callu,v,any,any,any,,a,a_gA'>
21	uMdAlani	unk	<fs af='uMdu,v,any,any,any,,Ali_ani,Ali_ani'>
22	korukuMtU	unk	<fs af='koru,v,any,any,any,,koVn_wU,koVn_wU'>
23	wama	unk	<fs af='wama,avy,,,,,0,0_avy'>|<fs af='wAmu,pn,any,sg,2,o,ti,ti'>
24	kqwajFawalu	unk	<fs af='kqwajFawa,n,,pl,,d,0,0'>|<fs af='kqwajFawa,n,,pl,,d,vu,vu'>
25	cAtukunnAru	unk	<fs af='cAtu,v,fm,pl,3,,koVn_A,koVn_A'>
26	.	unk	<fs af='.,punc,,,,,,'>
