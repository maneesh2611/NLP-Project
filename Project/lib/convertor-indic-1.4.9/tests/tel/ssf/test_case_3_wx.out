1	సూపర్	unk	<fs af='సూపర్,n,,sg,,d,0,0'>|<fs af='సూపర్,n,,sg,,o,టి,ti'>
2	స్టార్	unk	<fs af='స్టార్,n,,sg,,d,0,0'>|<fs af='స్టార్,n,,sg,,o,టి,ti'>
3	ననే	unk	<fs af='నన,n,,sg,,,0,0_e'>
4	అహం	unk	<fs af='అహం,n,,sg,,d,0,0'>
5	లేకుండా	unk	<fs af='లేకుండా,avy,,,,,0,0_avy'>|<fs af='లేకుండు,v,any,any,any,,ఇ_ఆ,i_A'>
6	ఆయన	unk	<fs af='ఆయన,pn,,sg,,d,0,0'>|<fs af='ఆయన,pn,,sg,,o,టి,ti'>
7	చూపిన	unk	<fs af='చూపు,v,any,any,any,,ఇన,ina'>
8	ఆప్యాయతకు	unk	<fs af='ఆప్యాయత,n,,sg,,,కి,ki'>
9	కదిలిన	unk	<fs af='కదులు,v,any,any,any,,ఇన,ina'>
10	పోయిన	unk	<fs af='పోయిన,n,,sg,,d,0,0'>|<fs af='పోయిన,n,,sg,,o,టి,ti'>|<fs af='పో,v,any,any,any,,ఇన,ina'>
11	యూనిట్	unk	<fs af='యూనిట్,n,,sg,,d,0,0'>|<fs af='యూనిట్,n,,sg,,o,టి,ti'>
12	సభ్యులు	unk	<fs af='సభ్యుడు,n,,pl,,d,0,0'>|<fs af='సభ్యుడు,n,,pl,,d,వు,vu'>
13	రజనీ	unk	<fs af='రజని,n,,sg,,,0,0_V'>
14	,	unk	<fs af=',,punc,,,,,'>
15	ఆయన	unk	<fs af='ఆయన,pn,,sg,,d,0,0'>|<fs af='ఆయన,pn,,sg,,o,టి,ti'>
16	కుటుంబ	unk	<fs af='కుటుంబం,n,,sg,,o,టి,ti'>
17	సభ్యులు	unk	<fs af='సభ్యుడు,n,,pl,,d,0,0'>|<fs af='సభ్యుడు,n,,pl,,d,వు,vu'>
18	పదికాలాలు	unk	<fs af='పదికాలాలు,unk,,,,,,'>
19	పాటు	unk	<fs af='పాటు,n,,sg,,d,0,0'>|<fs af='పాటు,n,,sg,,o,టి,ti'>
20	చల్లగా	unk	<fs af='చల్లగా,avy,,,,,0,0_avy'>|<fs af='చల్లగా,avy,,,,,0,0_avy'>|<fs af='చల్ల,n,,sg,,,గా,gA'>|<fs af='చల్ల,adj,,sg,,,గా,gA_adj'>|<fs af='చల్లు,v,any,any,any,,అ,a_gA'>
21	ఉండాలని	unk	<fs af='ఉండు,v,any,any,any,,ఆలి_అని,Ali_ani'>
22	కోరుకుంటూ	unk	<fs af='కోరు,v,any,any,any,,కొన్_తూ,koVn_wU'>
23	తమ	unk	<fs af='తమ,avy,,,,,0,0_avy'>|<fs af='తాము,pn,any,sg,2,o,టి,ti'>
24	కృతజ్ఞతలు	unk	<fs af='కృతజ్ఞత,n,,pl,,d,0,0'>|<fs af='కృతజ్ఞత,n,,pl,,d,వు,vu'>
25	చాటుకున్నారు	unk	<fs af='చాటు,v,fm,pl,3,,కొన్_ఆ,koVn_A'>
26	।	unk	<fs af='।,punc,,,,,,'>
